### DISCUSSION TITLE
Building issue [fontkit related]

### DISCUSSION THREAD

**Discussion by Sanjaydotcom** [diegomura/react-pdf#2412] [Q&A]
Hello everyone,

I'm currently facing a build issue with the react-pdf package. My environment is Node.js version 14.20.0 and npm version 6.14.17. While attempting to build the project, I encountered the following error message:

ERROR in ./node_modules/fontkit/dist/browser-module.mjs 13217:10-22
Can't import the named export 'uint8' from non EcmaScript module (only default export is available)

Has anyone else encountered this issue or does anyone have insights on how to resolve it? Your help would be greatly appreciated.

**Comment by geranton93** [comment]
If you are using webpack try to add this code to configs 
`resolve: {
    extensions: ['*', '.mjs', '.js', '.json']
  },
  module: {
    rules: [
      {
        test: /\.mjs$/,
        include: /node_modules/,
        type: 'javascript/auto'
      }
    ]`
